tools/status/report.sh

```diff
@@ -65,8 +66,11 @@ function write_job_log() {
 
 function pull_latest_job_logs() {
   # pull the logs for the latest ten jobs for this connector
+  # note this is done by key as each log has a timestamp in the filename
+  # ensuring that the version specific runs are filtered out.
   LAST_TEN_FILES=$(aws s3api list-objects-v2 --bucket "$BUCKET"  \
-    --query "reverse(sort_by(Contents[?contains(Key, \`tests/history/$CONNECTOR\`)], &LastModified))[:10].Key" \
+    --prefix "tests/history/$CONNECTOR" \
+    --query "reverse(sort_by(Contents[?!contains(Key, \`$VERSION_PREFIX\`)], &Key))[:10].Key" \
     --output=text)
```

Comment:
This query is done by the S3 API, yes?  That means we don't actually need to download the ever-growing list of files and do the sort locally?

Reply:
Yup!

Reply:
That also why this took a bit longer as I wanted to make s3 responsilbe for the final "take 10". Just so this wouldnt get slower and slower as more logs pile up